source/includes/code-examples/update-many/UpdateMany.cs

```diff
@@ -0,0 +1,86 @@
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace UsageExamples.UpdateMany
+{
+    public class UpdateMany
+    {
+        private static IMongoCollection<Restaurant> _restaurantsCollection;
+        private const string MongoConnectionString = "<Your MongoDB URI>";
+
+        private const string OldCuisine = "Pizza";
+        private const string NewCuisine = "Pasta and breadsticks";
+        private const string CuisineField = "cuisine";
+
+        public static void Main(string[] args)
+        {
+            Setup();
+
+            // Extra space for console readability 
+            WriteLine();
+
+            // Number of restaurants with old cuisine
+            WriteLine($"Restaurants with {CuisineField} \"{OldCuisine}\" found: {FindCountOfRestaurantsWithCuisine(OldCuisine)}");
+
+            // Update many documents synchronously
+            var syncResult = UpdateManyRestaurants();
+            WriteLine($"Restaurants modified by update: {syncResult.ModifiedCount}");
+
+            // Number of restaurants with new cuisine
+            WriteLine($"Restaurants with {CuisineField} \"{NewCuisine}\" found after update: {FindCountOfRestaurantsWithCuisine(NewCuisine)}");
+
+            // Reset sample data
+            Write("Resetting sample data...");
+            ResetSampleData();
+            WriteLine("done.");
+        }
+
+        private static UpdateResult UpdateManyRestaurants()
+        {
+            // start-update-many
+            const string oldValue = "Pizza";
+            const string newValue = "Pasta and breadsticks";
```

Comment:
My reasoning was that if you were looking at just the code, it would show what each method parameter represents. That way, if a reader wanted to adapt this sample for their own purposes, it would be clear more quickly where they needed to make changes. 

For comparison, here's the version without those constants:

```
private static UpdateResult UpdateManyRestaurants()
{
    var filter = Builders<Restaurant>.Filter
        .Eq("cuisine", "Pizza");

    var update = Builders<Restaurant>.Update
        .Set(restaurant => restaurant.Cuisine, "Pasta and breadsticks");

    return _restaurantsCollection.UpdateMany(filter, update);
}
```

Also happy to rope in the team to see what they prefer!

Reply:
I'm not sure I agree that the addition of the variables makes the code more easy to understand or reuse (and thus justifies the increased vertical scroll), but I'm alright resolving the conversation if you feel it improves the page. Just wanted to point it out to be reconsidered.

I don't think it makes sense to loop team in for this. Feel free to resolve this thread and implement whatever you think is best.